src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp/Components/Account/Pages/Manage/DeletePersonalData.razor

```diff
@@ -53,11 +53,19 @@
     {
         Input ??= new();
         user = await UserAccessor.GetRequiredUserAsync(HttpContext);
-        requirePassword = await UserManager.HasPasswordAsync(user);
+        if (user is not null)
+        {
+            requirePassword = await UserManager.HasPasswordAsync(user);
+        }
     }
 
     private async Task OnValidSubmitAsync()
     {
+        if (user is null)
+        {
+            throw new InvalidOperationException("User is not loaded; failed to delete personal data.");
+        }
```

Comment:
If `NavigateTo()` gets called during `InitializeAsync()`, does that now prevent form submit handlers from running? If not, I'm guessing we wouldn't want this method to throw when the user doesn't exist.

Reply:
(similar question for similar code in other places)